newdle/client/src/client.js

```diff
@@ -112,9 +112,13 @@ class Client {
     });
   }
 
-  getParticipant(newdleCode, participantCode) {
-    const params = {code: newdleCode, participant_code: participantCode};
-    return this._request(flask`api.get_participant`(params), {anonymous: true});
+  getParticipant(newdleCode, participantCode, me) {
```

Comment:
`me = false`?

Reply:
I specified that inside the `fetchParticipant` action. Should I specify it here too?

Reply:
I think it makes sense to keep defaults in all "public" places (and the client is one for sure)